modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file
+ * The iNope Policy Review Module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function inope_policy_review_menu() {
+  $items = array();
+
+  $items['node/%/review'] = array(
+    'title callback' => 'inope_policy_review_tab_title_callback',
+    'title arguments' => array(1),
+    'description' => 'Review of policy nodes.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('inope_policy_review_form'),
+    'access arguments' => array('view published content'),
+    'type' => MENU_LOCAL_TASK,
+  );
+
+  return $items;
+
+}
+
+/**
+ * The iNope Policy Review Page Title Callback.
+ */
+function inope_policy_review_tab_title_callback($node_nid) {
+  $node = node_load($node_nid);
+  $node_title = $node->title;
+  $title = t('Review of @node_title', array('@node_title' => $node_title));
+  return $title;
+}
+
+/**
+ * The iNope Policy Review Form.
+ */
+function inope_policy_review_form($form, &$form_state) {
+
+  $form['inope_policy_review_approved_checkbox'] = array(
```

Comment:
This is a bit weird. The states control over the checkbox is essentially making it behave like a single radio form control, unless there is something else you are doing here. Did you try a radio here instead and then make the other item conditional to the value selected?